docs/configuration/ai-analysis.rst

```diff
@@ -7,18 +7,12 @@ Robusta can integrate with `Holmes GPT <https://github.com/robusta-dev/holmesgpt
 
 Holmes can be installed with Robusta by adding ``enableHolmesGPT: true`` to the Robusta ``generated_values.yaml`` file.
 
-When available, AI based investigations can be launched from a button in Slack, or from the Robusta UI.
+When available, AI based investigations can be launched from the ``Ask Holmes`` button in Slack, or from the Robusta UI.
 
-When enabled, your Slack messages will have the ``Ask Holmes`` button:
-
-   .. image:: /images/pod-pending-alert.png
-       :width: 600px
-
-Clicking it, will launch a Holmes investigation, which will send the results back to Slack:
-
-   .. image:: /images/pending-alert-holmes-response.png
+   .. image:: /images/robusta-holmes-investigation.png
```

Comment:
Do you have an indentation here which leads to the image being block quoted?

![Screenshot 2024-07-01 at 11 13 43](https://github.com/robusta-dev/robusta/assets/494087/e1909b04-ba95-4715-be9f-cf96b3de6938)

Reply:
Fixed